R/readBindingConstraints.R

```diff
@@ -44,76 +72,251 @@ readBindingConstraints <- function(opts = simOptions()) {
     }
   }
   
+  ##
+  # API BLOC
+  ##
+  
   if(opts$typeLoad == 'api'){
-    bindingConstraints <- read_secure_json(file.path(opts$inputPath, "bindingconstraints", "bindingconstraints"), 
-                                           opts$token, timeout = opts$timeout, config = opts$httr_config)
+    bindingConstraints <- read_secure_json(file.path(opts$inputPath, 
+                                                     "bindingconstraints", 
+                                                     "bindingconstraints"), 
+                                           opts$token, 
+                                           timeout = opts$timeout, 
+                                           config = opts$httr_config)
   }else{
-    path <- file.path(opts$inputPath, "bindingconstraints/bindingconstraints.ini")
-    bindingConstraints <- readIniFile(path, stringsAsFactors = FALSE)
+    path <- file.path(opts$inputPath, 
+                      "bindingconstraints/bindingconstraints.ini")
+    bindingConstraints <- readIniFile(path, 
+                                      stringsAsFactors = FALSE)
   }
   
+  ##
+  # Exception if no properties
+  ##
+  
   if(length(bindingConstraints) == 0) {
     warning("It looks like there is no binding constraints is this study.")
```

Comment:
Eviter le conditionnel et envoyer un warning du style "No binding constraints found in this study" ?